docker-compose.yml

```diff
@@ -17,6 +17,9 @@ services:
          - TZ=UTC
          - ADMIN_PASSWORD="adminPasswordHere"
          - COMMUNITY=false  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
+         - BACKUP_ENABLED=false
```

Comment:
Backups are enabled by default in the Dockerfile. Here you set it to false, what is the preferred status of this setting?